datasets/us/plum_book/crawler.py

```diff
@@ -0,0 +1,54 @@
+import csv
+
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+IGNORE = [
+    "OrganizationName",
+    "PositionStatus",
+    "PaymentPlanDescription",
+    "LevelGradePay",
+    "Tenure",
+]
+
+
+def crawl(context: Context):
+    path = context.fetch_resource("source.csv", context.data_url)
+    with open(path, "r", encoding="utf-8") as fh:
+        reader = csv.DictReader(fh)
+        for row in reader:
+            agency_name = row.pop("\ufeffAgencyName")
+            position_title = row.pop("PositionTitle", None)
+            appointment_type = row.pop("AppointmentTypeDescription")
+            expiration_date = row.pop("ExpirationDate")
+            location = row.pop("Location")
+            incumbent_first_name = row.pop("IncumbentFirstName")
+            incumbent_last_name = row.pop("IncumbentLastName")
+
+            if not incumbent_first_name or not incumbent_last_name:
+                continue
+            person = context.make("Person")
+            person.id = context.make_slug(incumbent_first_name, incumbent_last_name)
+            h.apply_name(
+                person, first_name=incumbent_first_name, last_name=incumbent_last_name
+            )
+            person.add("position", f"{position_title}, {agency_name}")  # for dedupe
+
+            position = h.make_position(
+                context,
+                name=f"{position_title}, {agency_name}",
+                description=appointment_type,
```

Comment:
I think appointment type technically belongs on the occupancy, not the position. There are probably easier ways to determine this, but here are examples where one position name would come up with multiple appointment types. That means one Position entity would end up with all three of these in its description property.

```
grep '"Position"' data/datasets/us_plum_book/entities.ftm.json | jq '(.properties.description | length | tostring) + " " + .properties.name[0]' | sort -n
...
"3 SENIOR ADVISOR, DEPARTMENT OF THE INTERIOR"
"3 SENIOR ADVISOR, OFFICE OF PERSONNEL MANAGEMENT"
"3 SENIOR ADVISOR, OFFICE OF THE SECRETARY OF DEFENSE"
"4 SENIOR ADVISOR, DEPARTMENT OF HOUSING AND URBAN DEVELOPMENT"
```